platform/discord/manifest.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: discordbot-service
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: pocketsizefund/discordbot:202409.4.0
+          ports:
+            - containerPort: 8080
+          env:
+            - name: DISCORD_WEBHOOK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: DISCORD_WEBHOOK_URL
+---
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: vibe-check
+spec:
+  schedule: "15 9 * * *"
+  timezone: "America/New_York"
+  data: 'gm quants, lets attack the day'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: discordbot-service
+---
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: vibe-check
+spec:
+  schedule: "15 9 * * *"
+  timezone: "America/New_York"
+  data: 'no matter what is happening you are awesome'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: discordbot-service
```

Comment:
This one can probably be removed, right?

Reply:
vibe check stays